upgrade/upgrade-scripts/execute

```diff
@@ -604,7 +604,13 @@ if [[ -f "$UPDATE_DIR/upgrade.properties" ]]; then
 	source_upgrade_properties
 fi
 
-if [[ -n "$opt_f" ]] || [[ "$UPGRADE_TYPE" == "FULL" ]]; then
+#
+# On versions 17.0 and greater, we don't issue the reboot. Rather,
+# we restart the delphix services, and the reboot will be issued by
+# the virtualization service as it starts up.
+#
+if { [[ -n "$opt_f" ]] || [[ "$UPGRADE_TYPE" == "FULL" ]]; } &&
+	compare_versions "$CURRENT_VERSION" lt "17.0.0.0-0"; then
```

Comment:
Shouldn't this be 18.0.0.0 now that `develop` points to 18.0 currently? Or is the plan to also backport this to 17.0?

Reply:
yes, you're right, I'll update it.. and in the app-gate too as needed..